.travis.yml

```
@@ -2,17 +2,16 @@ language: julia
 
 os:
   - linux
+  - windows
   - osx
 
 julia:
-  - 1.0
   - 1.3
-  - nightly
 
-before_script:
-  # Every 30 seconds, look for the build log file. If it exists, then
-  # start watching its contents and printing them to stdout as they
-  # change. This has two effects:
-  #   1. it avoids Travis timing out because the build outputs nothing
-  #   2. it makes it more obvious what part of the build, if any, gets stuck
-  - while sleep 30; do tail ./deps/build.log -f ; done &
+notifications:
+  email: false
+
+script:
+    julia --project="" -e 'using Pkg; Pkg.add(PackageSpec(name ="PackageCompiler", rev="master"))'; 
```

```suggestion
    julia --project="" -e 'using Pkg; Pkg.add("PackageCompiler")'; 
```
Now that PackageCompiler is registered this can be updated